src/analytics/PivotTable.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import PivotTableUI from 'react-pivottable/PivotTableUI';
+import 'react-pivottable/pivottable.css';
+
+const PivotTable = ({ cards }) => {
```

Comment:
Might be good to add a description for those of us who don't know how to use a pivot table.

Reply:
It's a somewhat advanced analytic tool, I'll link to a video describing how to use pivottables in general but I don't think it's something we could feasibly explain here